Azure/communication-ui-library-ios
# [Bugfix] Fix up chat signing and enable builds in release mode

## Purpose
The migration of chat to develop needed some build job changes.
The test targets were also set to the enterprise team.
Additionally, the calling sdk mock changes needed some modifications to compile in release mode where test imports are not available.

## Does this introduce a breaking change?


## Pull Request Type
What kind of change does this Pull Request introduce?


## Verification Steps
Build a debug and release demo app; note no compilation problems.
See build here 
https://skype.visualstudio.com/SCC/_build/results?buildId=52605371&view=results
